src/components/Articles.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import { Link } from "react-router-dom";
+import "./Articles.css";
+
+const instance = axios.create({
+  baseURL: "https://news-945o.onrender.com/api/",
```

Comment:
Great job with this! Ideally we could also setup a file called api.js for example where we keep "instance" but also your instance.get wrapped in a function, at that point inside the component you would only invoke the function that you created 🙂